packages/node/src/indexer/dictionary.service.test.ts

```diff
@@ -31,9 +31,8 @@ describe('DictionaryService', () => {
   it('should return all specVersion', async () => {
     const project = testSubqueryProject();
     const dictionaryService = new DictionaryService(project, nodeConfig);
-
-    const specVersions = await dictionaryService.getSpecVersions();
-
-    expect(specVersions.length).toBeGreaterThan(0);
+    const specVersions = dictionaryService.getSpecVersions();
+    expect.assertions(1);
+    return specVersions.then((res) => expect(res.length).toBeGreaterThan(0));
```

Comment:
I would prefer we use await but I don't understand whats going on here. Are you able to explain please?